### ISSUE TITLE
adding item

### ISSUE THREAD

**Issue by BraianLDC** [Noxde/Bloodborne-save-editor#13]
First off, Thank you for this amazing Tool, its truly a blessing

Would it be possible to allow for item additions?, currently can only replace existing items

also firing hammer badge and yharnam stone are not unlocking items on messenger shop

**Comment by Noxde** [comment]
Thanks you for trying it out, we're glad that you liked it!

It is possible to add items but I don't really like it because it doesn't work as expected, after adding items you would need to buy new items that you dont already have in your inventory for them to show up. (or at least I haven't found another way of making it work) we can add it on the next release and have a little guide or something though.

I'll take a look into those not working properly.

Thanks for your feedback!

**Comment by BraianLDC** [comment]
I understand, asking mostly because the only way to get other armor items right now is only by replacing the ones you have

**Comment by Noxde** [comment]
Do you have the dlc? I tried replacing something for the firing hammer badge and I didn't unlock anything either but I don't have the dlc, and its possible there is a flag to check if the yharnam stone should unlock anything too. It might be that the yharnam stone actually doesn't unlock anything and it actually unlocks something when you kill the queen

**Comment by BraianLDC** [comment]
yes, i tested it just now, even accessing the DLC and facing the enemy,  you wont get any item unlocks until you actually kill the NPC that drops the firing hammer badge

**Comment by Noxde** [comment]
It must use a flag to unlock the items then, it would be possible to set the flag but I'm honestly not sure how to properly look for them, there are a lot of things you could do by setting different flags like unlock lanterns and things like that.

**Comment by BraianLDC** [comment]
yea probably the yharnam stone is working the same way

**Comment by Noxde** [comment]
Adding items introduced in [0.5.0](https://github.com/Noxde/Bloodborne-save-editor/releases/tag/0.5.0)
